wgengine/netstack/netstack.go

```diff
@@ -880,7 +943,7 @@ var isSynology = runtime.GOOS == "linux" && distro.Get() == distro.Synology
 //
 // TODO(bradfitz): when we're running on Windows as the system user, use
 // raw socket APIs instead of ping child processes.
-func (ns *Impl) userPing(dstIP netip.Addr, pingResPkt []byte) {
+func (ns *Impl) userPing(dstIP netip.Addr, pingResPkt []byte, direction userPingDirection) {
```

Comment:
document this parameter? especially how it related to the other two parameters.

Reply:
Added a comment, and also a test while I was in there; PTAL?